bin/install-nplinker-deps

```diff
@@ -125,6 +125,7 @@ pip install -q -U pip setuptools
 echo "🔥 Start installing BigScape ..."
     [[ -d BiG-SCAPE ]] || git clone https://github.com/medema-group/BiG-SCAPE.git
     cd BiG-SCAPE
+    git reset --hard
```

Comment:
I added this during testing of installation and forgot to remove it.

However, this does ensure that when you re-run install-nplinker-deps, all tasks complete successfully.
Otherwise, this script crashes here because you first need to stash/commit/revert changes before you do the other steps.

Would it be practical to just skip all of this entirely if the BiG-SCAPE directory already exists?

Reply:
Interesting, I did not see any changes to the local code repo. Why is it necessary to stash/commit/revert **changes** before you do the other steps? Is there anything I miss?

Reply:
The modification of permissions here:

```bash
    chmod 754 bigscape.py
    chmod 664 domains_color_file.tsv
    chmod 775 Annotated_MIBiG_reference
```

Is being tracked and results in conflicts on a re-run for me.

Reply:
Aha, indeed! OK, then we can keep the `git reset --hard` to solve the local changes problem.